packages/react/src/BlockNoteTheme.ts

```diff
@@ -246,22 +325,24 @@ export const getBlockNoteTheme = (
                 backgroundColor: hoveredBackground,
               },
             },
+            ...componentStyles?.Toolbar,
```

Comment:
The difficulty now of course is that if I want to change sth to `.mantine-UnstyledButton-root`, it will override the stuff from line 290. I suppose there's no easy way to prevent this? (again, let's not spend too much time on this)

Reply:
We would have to deep merge the existing styles with the new styles, I think theres a function from lodash actually which might do that.